### ISSUE TITLE
Open Squeezer from webpage

### ISSUE THREAD

**Issue by bennyslbs** [kaaholst/android-squeezer#823]
I would like to open Squeezer from a link on a webpage or from HomeAssistant app.

Like issue #137 but not via ADB but via link on a web-page.

There is a mini HOWTO showing different actions: https://github.com/kaaholst/android-squeezer/issues/137#issuecomment-770409328

Is it possible to open now playing NowPlayingActivity - the other does not work for me.
I would like to open the default/main page with Music/Radio/Search/... menu. Prefferable for a specific player.

Is that possible?

**Comment by kaaholst** [comment]
I don't know how that works. There seem to be information here: https://developer.android.com/training/app-links/
How do you manage to open NowPlayingActivity?

**Comment by bennyslbs** [comment]
I open NowPlayingActivity from adb.
I think the link you shared is link from a app. I want link from webpage to app.

I thought it was just me there couldn't figure out how to do it, but seems not so easy today.
I will close the issue. Sorry for interrupting you.
Squeezer is great, thank you!

I have found:
https://www.branch.io/resources/blog/how-to-open-an-android-app-from-the-browser/
This requires verification of app and website with link to app.

Prior I found this, but it is 11 years old, so that propably not possible today:
https://stackoverflow.com/questions/2958701/launch-custom-android-application-from-android-browser
